code/script.js

```diff
@@ -225,101 +232,164 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person selected')
+  console.log(secret)
 }
 
 // This function to start (and restart) the game
 const start = () => {
+  console.log('start funktion is called!')
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+  // set the secret person -> invoke set Secret funktion
+  setSecret()
+  //show the board on the screen -> invoking generateboard funktion. 
+  generateBoard()
 }
```

Comment:
You could revise` start() `to include the few things from newGame, and remove newGame. It is the same thing, ie drawing up the board of characters, selecting the secret character, and setting the counter to 0. So if you add the two lines plus set count to 0, the event listener on `playAgainButton` can call start. 
```javascript 
winOrLose.style.display = 'none'
board.style.display = 'flex' `
count = 0 // add here to reset your question counter

Reply:
thanks, i updated the start function as you suggested and removed the new game function. Also tried to clear the counter but don't get it to work. Any ideas on how to solve this?

Reply:
I just tested added this line of code to your start function after you set `count = 0` and it worked!
```javascript
guessCounter.innerHTML = `<p> Guesses made: ${count}</p>`
```
Otherwise, your counter variable is being reset to 0 when you start or restart, but the text being displayed isn't resetting until you submit a question. Hope that helps :)

Reply:
yes, it works! Thanks for the help :)